packages/office-ui-fabric-react/src/components/ColorPicker/ColorSlider.tsx

```diff
@@ -65,12 +66,12 @@ export class ColorSlider extends React.Component<IColorSliderProps, IColorSlider
     return (
       <div
         ref='root'
-        className={ css('ms-ColorPicker-slider', className, {
+        className={ css('ms-ColorPicker-slider', styles.slider, className, {
           'is-adjusting': isAdjusting
         }) }
         onMouseDown={ this._onMouseDown }>
-        <div className='ms-ColorPicker-sliderOverlay' style={ overlayStyle } />
-        <div className='ms-ColorPicker-thumb is-slider' style={ { left: currentPercentage + '%' } } />
+        <div className={ css('ms-ColorPicker-sliderOverlay', styles.sliderOverlay) } style={ overlayStyle } />
+        <div className={ css('ms-ColorPicker-thumb is-slider', styles.thumb, styles.isSlider) } style={ { left: currentPercentage + '%' } } />
```

Comment:
wait... I think this is unnecessary. this seems unnecessarily verbose. maybe this is a bad idea. You missed some above.